src/components/shared/StackSettings.js

```diff
@@ -214,37 +159,37 @@ class StackSettingsImpl extends PureComponent<Props> {
               </label>
             </li>
           ) : null}
-          {hideInvertCallstack ? null : (
-            <li className="stackSettingsListItem">
-              <label className="photon-label photon-label-micro stackSettingsLabel">
-                <input
-                  type="checkbox"
-                  className="photon-checkbox photon-checkbox-micro stackSettingsCheckbox"
-                  onChange={this._onInvertCallstackClick}
-                  checked={invertCallstack}
-                />
-                <Localized
-                  id="StackSettings--invert-call-stack"
-                  attrs={{ title: true }}
-                >
-                  <span>Invert call stack</span>
-                </Localized>
-              </label>
-            </li>
-          )}
-          {selectedTab !== 'stack-chart' ? null : (
-            <li className="stackSettingsListItem">
-              <label className="photon-label photon-label-micro stackSettingsLabel">
-                <input
-                  type="checkbox"
-                  className="photon-checkbox photon-checkbox-micro stackSettingsCheckbox"
-                  onChange={this._onShowUserTimingsClick}
-                  checked={showUserTimings}
-                />
-                <Localized id="StackSettings--show-user-timing">
-                  Show user timing
-                </Localized>
-              </label>
+          {hideInvertCallstack && selectedTab !== 'stack-chart' ? null : (
+            <li className="panelSettingsListItem">
```

Comment:
I combined these 2 options into the same list item, so that they don't have a border in-between. I considered keeping it initially but then decided to go with this solution.